README.md

```diff
@@ -1,2 +1,88 @@
-# sncosmo-eps
-SNCosmo Enhancement Proposals
+# SNCosmo Enhancement Proposals
+ 
+EPs are documents to address non-trivial enhancements that require discussion and thought 
+beyond a single Pull Request. This is intended to mirror the long-standing Python Enhancement 
+Proposal process and Astropy's Proposals for Enhancement, but generally not quite as
+formally as those much larger projects. Normally a proposal goes through various phases of 
+consideration. Discussion is expected to take place using existing mechanisms (github, mailing 
+list, etc), and eventually a decision is made regarding whether the proposal should be accepted, 
+rejected, or tabled for further discussion later.
+
+## Accepted EPs
+
+| # |     Title                          | Date        | DOI   |
```

Comment:
I think we can leave DOIs off. Like you write in the intro, this is a more informal process than astropy's. Let's keep it as lightweight as possible until we see a need for something more.